tests/src/common.rs

```diff
@@ -0,0 +1,61 @@
+use serde:: {Serialize, Deserialize};
+use anyhow::Result;
+use rand::{distributions::Alphanumeric, Rng};
+
+pub async fn send_get(
+    url: &str
+) -> Result<reqwest::Response, reqwest::Error> {
+    let client  = reqwest::Client::new();
+    let response = client
+    .get(url)
+    .header("Content-Type", "application/json")
+    .send().await;
+
+    response
+}
+
+pub async fn send_post(
+    url: &str,
+    payload: String
+) -> Result<reqwest::Response, reqwest::Error> {
+    let client = reqwest::Client::new();
+    let response = client
+        .post(url)
+        .body(payload)
+        .header("Content-Type", "application/json")
+        .send()
+        .await;
+
+    response
+}
+
+pub async fn random_payload() -> String {
```

Comment:
nit: might want to change this function name? this is not returning random payload, it's returning a valid json with random order details.

Reply:
also order detail are a payload ;-)